corehq/apps/app_manager/views/notifications.py

```diff
@@ -0,0 +1,41 @@
+import json
+import datetime
+from django.utils.translation import ugettext as _
+from ws4redis.publisher import RedisPublisher
+from ws4redis.redis_store import RedisMessage
+from corehq.toggles import APP_BUILDER_NOTIFICATIONS
+from dimagi.utils.parsing import json_format_datetime
+
+
+def notify_form_opened(domain, couch_user, app_id, unique_form_id):
+    message = _('This form has been opened for editing by {}.').format(couch_user.username)
+    notify_event(domain, couch_user, app_id, unique_form_id, message)
+
+
+def notify_form_changed(domain, couch_user, app_id, unique_form_id):
+    message = _(
+        'This form has been updated by {}. Reload the page to see the latest changes.'
```

Comment:
I've seen other sites with this functionality have the `Reload` word be a link back to the page itself (or some javascript that reloads the page). Might be nice to add that at some point.